README.md

```diff
@@ -265,11 +267,12 @@ To see the full-color output of a specific integration test, you can use `PRINT_
 
 ## Releasing a new version (Shopify employees)
 
-1. Update the version number in `version.rb` and commit that change alone to master.
-2. Tag the version with `git tag vx.y.z && git push --tags`
-3. Use the [Shipit Stack](https://shipit.shopify.io/shopify/kubernetes-deploy/rubygems) to build the `.gem` file and upload to [rubygems.org](https://rubygems.org/gems/kubernetes-deploy).
+1. Update the version number in `version.rb` and commit that change with message "Version x.y.z". Don't push yet or you'll confuse Shipit.
+2. Tag the version with `git tag vx.y.z -a -m "Version x.y.z"`
```

Comment:
We're using Shipit though. Nobody should do `rake release` locally anymore. Only you and I still have the permissions to do so.